src/cantools/database/can/__init__.py

```diff
@@ -1,5 +1,7 @@
-from .bus import Bus
-from .database import Database
-from .message import DecodeError, EncodeError, Message
-from .node import Node
-from .signal import Signal
+from .bus import Bus as Bus
+from .database import Database as Database
+from .message import DecodeError as DecodeError
+from .message import EncodeError as EncodeError
+from .message import Message as Message
+from .node import Node as Node
```

Comment:
I checked csource.py with `mypy --strict` and it complained about that file. These names are exported according to PEP 484:

> Modules and variables imported into the stub are not considered exported from the stub unless the import uses the import ... as ... form or the equivalent from ... import ... as ... form. (UPDATE: To clarify, the intention here is that only names imported using the form X as X will be exported, i.e. the name before and after as must be the same.)

Alternatively, we could put them in `__all__` if you like.

Reply:
Ok. I just think that this is a quite clumsy construct. does mypy complain without `--strict`?

Reply:
I replaced it with `__all__`. That works too.

Reply:
Now i removed `__all__`, because it causes issues with these star imports:

```python3
# Remove once less users are using the old package structure.
from .can import *  # noqa: F403
```